internal/clients/http/gcp/gcp_client.go

```diff
@@ -0,0 +1,91 @@
+package gcp
+
+import (
+	"context"
+	"fmt"
+
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+
+	compute "cloud.google.com/go/compute/apiv1"
```

Comment:
One question tho - aren't you using too low-level API? That looks like some protobuf layer. I need to read some examples about this.

Reply:
After reading some examples, it really looks like this is it. Their API is heavily based around gRPC.